bundles/org.openhab.binding.shelly/doc/images/UseCaseSmartRoller.md

```diff
@@ -0,0 +1,225 @@
+- INPUT BUTTONS MODE: This depend on your installed button, if you only have a single one use "One button", if you have 2 buttons or one with 2 positions select "One button"
+- POSITIONING CONTROLS: Enable them to have the controls in the App (UP/STOP/DOWN + relative positions), this helps during installation
+
+
+For safety reasons you might consider:
+- OPEN/CLOSE WORKING TIME
+This feature makes sure that the Shelly stops the motor after a certain time even the motor doesn't stop by itself. 
+Start the roller, measure the number of seconds until the shutter stops.
+Maybe add 2-3 seconds for safe and set the values for both directions in the UI
+
+![](images/uiroller_wt.png)
+
+- OBSTACLE DETECTION under Safety in the Web UI
+Shelly 2.5 has an integrated detection on roller obstacles based on measuring the motor power.
+The detection needs to be enabled, "While Moving" cover open and close.
+Action Stop makes most sense, you should manually verify the problem.
+
+![](images/uiroller_obs1.png)   ![](images/uiroller_obs2.png)      ![](images/uiroller_obs3.png)
+
+and there are some more options:
+- MAX POWER PROTECTION
+A roller should never consume high power - in this case you could assume an electrical problem.
+Therefore you could configure the Shelly to turn off power when exceeding a certain limit (e.g.250W - value depend on your roller motor).
+
+- SWAP INPUTS and REVERSE DIRECTIONS - you could flip operation mode for open/close, this could help in non-standard operations, but could also indicate a wiring problen
+It also makes sense to give the device a name. Go to Settings:DEVICE NAME and select a symbolic name.
+This name will also discovered by the binding and added as a suffix to the thing name in the Inbox.
+It simplifies to identify the roller when you have a ton of Shelly things (believe me).
+
+### Shelly setup - Positioning Favorites
+
+With version 1.9 Alterco introduced an interesting feature called Favorites.
+Those allow to store up to 4 pre-defined positions in the device, e.g. 15%, 50%, 65%, 98%.
+Once defined you have kind of a short-cut to bring the roller to that position and those favorites are also supported by the binding (see below).
+
+![](images/uiroller_fav1.png)   ![](images/uiroller_fav2.png)
+   
+To use this feature open the device's Web UI and click on Favorites(+) button.
+You could now specify 4 percentage values between 0% (closed) and 100% (open).
+Enter the values and try using the button line above the roller control and main UI buttons.
+
+With this configuration your Shelly is ready to be integrated into openHAB.
+
+### Binding/Thing configuration
+
+Once the Shelly is configured you could start the discovery within openHAB.
+Important: If you want to password protect your Shellys (strong recommendation) you could set a default user/password in the binding settings and avoid to specify those per thing:
+- Open PaperUI:Configuration:Binding
```

Comment:
Can you express this more abstract, as PaperUI will be removed in OH3?

Reply:
changed to "Open the Binding Configuration in the openHAB UI"